app/src/main/java/org/jellyfin/androidtv/util/NumberExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package org.jellyfin.androidtv.util
+
+import org.jellyfin.androidtv.TvApp
+
+/**
+ * Current (pixel) value as display pixels
+ */
+val Int.dp: Int
```

Comment:
This seems to be doing the same thing as this [method](https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/util/Utils.java#L69-L72).

Reply:
I actually introduced this change in another PR (https://github.com/jellyfin/jellyfin-androidtv/pull/307/commits/1e1d83b487ad748bcbfb53bfdb36113aaafe747d#diff-3d7ec439941e8f132c864ebd3a25191e). I didn't know we already had a method to do this calculation.

This variant uses a Kotlin extension function so we can write `7.dp` and it will convert the integer into pixels (for my emulator it would return `14`). I think it looks better than `Utils.convertDpToPixel(context, 7)`.

It's not really possible to use the extension function in the Java classes so I changed the extension function to use the Utils class instead.